anyric/super-record
# #167230537 Generate User Register

#### What does this PR do?
This PR create a view to generate and display user register.

#### Description of Task to be completed?
- Generate users report and display in the users template

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Login to the system
- Navigate to `http://127.0.0.1:8000/settings` and select `Users` menu item to see the list of users

#### Any background context you want to provide?
Currently admin can only see the registered users in the system from the default Django admin console. This is not the recommended, practice. There should be a report page under setting where an admin can see all registered users and there roles in one convenient place.

#### What are the relevant pivotal tracker stories?
[#167230537](https://www.pivotaltracker.com/story/show/167230537)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/61189319-6e745880-a694-11e9-85ea-1508af608edb.png)

#### Questions:
N/A
